model_compression_toolkit/exporter/model_wrapper/pytorch/validate_layer.py

```diff
@@ -31,12 +34,35 @@ def is_pytorch_layer_exportable(layer: Any) -> bool:
         Returns:
             Check whether a PyTorch layer is a valid exportable layer or not.
         """
+        if not isinstance(layer, nn.Module):
+            Logger.error(f'Exportable layer must be a Pytorch layer, but layer {layer.name} is of type {type(layer)}') # pragma: no cover
```

Comment:
Exportable layer must be a Pytorch *layer* -> Exportable layer must be a Pytorch *nn.Module*